src/main/java/gov/cancer/pageobject/crosscutting/LeadImage.java

```diff
@@ -0,0 +1,59 @@
+package gov.cancer.pageobject.crosscutting;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import gov.cancer.framework.ElementHelper;
+import gov.cancer.pageobject.PageObjectBase;
+
+public class LeadImage extends PageObjectBase {
+
+  @FindBy(how = How.CSS, using = "#cgvBody")
+  WebElement pageBody;
+
+  /********* LEAD IMAGE SELECTORS ***********************/
+
+  final public String imagecontainer = "div#cgvBody > div > figure";
+  final public String imagecaption = "div.caption-container p";
+  final public String imagecredit = ".image-photo-credit";
+  final public String imagediv = "div[class='centered-element'] img";
+
+  /**
+   * Constructor
+   *
+   * @param path
+   *          server-relative path of the page to load.
+   */
+  public LeadImage(String path) {
+    super(path);
+
+  }
+
+  /* Returns true if Lead image is displayed on the page */
+  public boolean isLeadImageVisible() {
+    return ElementHelper.isVisible(getBrowser(), imagecontainer);
+  }
+
+  /* Returns the caption of the image */
+  public String getCaption() {
+    return ElementHelper.findElement(getBrowser(), imagecaption).getText();
+  }
```

Comment:
This has come up before. It would likely be useful to add a `getText()` method to `ElementHelper`. 

```java
String getText(SearchContext context, String selector) {
    WebElement element = findElement(context, selector);
    if (element != null) {
      return element.getText();
    } else {
      return null;
    }
}
```

The idea being to get you out of writing the same code over again and just go straight to getting the value.